flow/src/org/labkey/flow/controllers/well/showWell.jsp

```diff
@@ -348,10 +348,10 @@ Ext.onReady(function()
     }
 });
 
-var statistics = <%=jsonStats%>;
+var statistics = <%=unsafe(jsonStats.toString())%>;
 var treeData;
 var stats;
-var keywords = <%=jsonKeywords%>;
+var keywords = <%=unsafe(jsonKeywords.toString())%>;

```

Comment:
No change needed here... but we really should make unsafe() and h() accept CharSequence instead of String. That would eliminate the need for these toString() calls. Soon...